default.nix

```diff
@@ -18,8 +18,9 @@ in
 , withDocs  ? true
 , withIde   ? false
 , withHadrianDeps ? false
-, withDwarf ? nixpkgs.stdenv.isLinux  # enable libdw unwinding support
-, withNuma  ? nixpkgs.stdenv.isLinux
+, withDwarf  ? nixpkgs.stdenv.isLinux  # enable libdw unwinding support
+, withNuma   ? nixpkgs.stdenv.isLinux
+, withDtrace ? nixpkgs.stdenv.isLinux
```

Comment:
Not darwin?

Reply:
Nope, I have no means of testing Darwin. If someone else wants to figure our the Darwin situation they should feel free ;)

Reply:
At least `linuxPackages.systemtap` isn't available under Darwin.

I've never used DTrace for GHC, so I don't know what's really needed. But if you've something to test, I can easily start a `nix-shell` and try it out... 😄

Reply:
BTW, there is `darwin.dtrace` in nixpkgs and it looks like Darwin is shipped with `dtrace` (`/usr/sbin/dtrace`).

Reply:
To be honest, I'm somewhat suspect that GHC's DTrace support has bit-rotted. Nevertheless, it does build with this change. You don't need to do anything in particular; `configure` automatically picks up that the `dtrace` executable is available.